src/renderers/shared/fiber/ReactFiberClassComponent.js

```diff
@@ -31,30 +35,36 @@ var invariant = require('invariant');
 const isArray = Array.isArray;
 
 module.exports = function(
-  scheduleSetState: (fiber : Fiber, partialState : any) => void,
-  scheduleReplaceState: (fiber : Fiber, state : any) => void,
-  scheduleForceUpdate: (fiber : Fiber) => void,
-  scheduleUpdateCallback: (fiber : Fiber, callback : Function) => void,
+  scheduleUpdate : (fiber : Fiber, priorityLevel : PriorityLevel) => void,
+  getPriorityContext : () => PriorityLevel,
 ) {
 
   // Class component state updater
   const updater = {
     isMounted,
-    enqueueSetState(instance, partialState) {
+    enqueueSetState(instance, partialState, callback) {
       const fiber = ReactInstanceMap.get(instance);
-      scheduleSetState(fiber, partialState);
+      const priorityLevel = getPriorityContext();
```

Comment:
I was going to remove `scheduleSetState` et al from the scheduler anyway because they were just an extra layer of indirection around `addUpdate` et al. See @sebmarkbage's comment here for context: https://github.com/facebook/react/pull/8538#discussion_r92308098

To avoid introducing this problem again we could delete `addCallback` since it's no longer used. Sebastian thought it was better not to break the existing class updater API on the off chance someone was relying on it.